easybuild/easyconfigs/o/Octave/Octave-4.2.1-intel-2017a-mt.eb

```diff
@@ -28,7 +32,7 @@ dependencies = [
     ('X11', '20170314'),
     ('PCRE', '8.40'),
     ('ncurses', '6.0'),
-    ('libreadline', '6.3'),
+    ('libreadline', '7.0'),
```

Comment:
@tutufan We usually don't fiddle with existing dependencies because that may result in introducing conflicts elsewhere.
The reason that this conflict popped up is because of the `Python` dependency of `PostgreSQL`.
We can avoid this by using a `PostgreSQL` dependency that doesn't have `Python` bindings (and thus no `Python` dependency).

Is `PostgreSQL` strictly needed for the `io` extension btw? It built fine for me without including `PostgreSQL`, but maybe that's because we have `postgresql-libs` installed in our system...

Reply:
It's been a while, but I believe `PostgreSQL` is required.  I almost certainly chose this `PostgreSQL` because it's the only reasonably up-to-date one.  It looks to me like the only non-Python one is several years old.  Are you thinking we should create a new one without Python, or am I missing something?

Reply:
Yes, I think adding a recent Python-less `PostgreSQL` and using that as a dependency here is the way forward.